dts/bindings/dma/nxp,mcux-edma-v2.yaml

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2020, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP MCUX EDMA version 2 controller
+
+compatible: "nxp,mcux-edma-v2"
+
+include: dma-controller.yaml
+
+properties:
+  reg:
+    required: true
+    description: |
+      Specifies base physical address(s) and size of DMA and respective DMAMUX register(s)
+      that routes DMA sources
+
+  interrupts:
+    required: true
+
+  dma-channels:
+    required: true
+
+  dma-requests:
+    required: true
+
+  dmamux-reg-offset:
+    type: int
+    default: 0
+    description:
+      The offset value for obtaining DMAMUX register index from DMAMUX channel.
+      Default value means DMAMUX channel is identical with DMAMUX register index
+
+  channel-gap:
+    type: array
+    description: |
+      On some platforms, there may be a gap in the channels and
+      this array specifies the start and end of a single gap
+
+  nxp,mem2mem:
+    type: boolean
+    description: If the DMA controller supports memory to memory transfer
+
+  nxp,a_on:
+    type: boolean
+    description: If the DMA controller supports always on
+
+  interrupt-ranges:
+    type: array
+    required: true
+    description: |
+      Describes range of channels for each interrupt id. For each interrupt id,
+      a couple of integers is provided: starting channel and offset between each
+      channel. As example:
+        interrupt-ranges = <0 0>, <1 0>, <2 0>, <3 0>,
+                          <4 0>, <5 0>, <6 0>, <7 0>,
+                          <8 0>, <9 0>, <10 0>, <11 0>,
+                          <12 0>, <13 0>, <14 0>, <15 0>,
+                          <0 1>;
+      Above property provides range for 17 interrupt ids.
+      16 first elements indicate each interrupt is for only one channel.
+      The last element indicates this interrupt is shared for all channels,
+      0 is starting channel and offset 1 between each.
+
+  no-error-irq:
```

Comment:
if this is IP related, I suppose it is for EDMA_V3, if this is true, I would not recommend adding the no-error-irq here, but add to v3

Reply:
i've updated